### PR TITLE
completed image component

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Image component

### CODE REVIEW

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    margin: 2rem;
+    width: auto;
+    border-radius: 0.2rem;
+}
+
+.img--square{
+ width: 350px;
```
Comment (AshutoshM29) [inline image/image.css]: You should use rem instead of px
Rest looks good!!

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]

File: image/image.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply image/image.css]: done

File: image/image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import url("/pages/docs.css");
+/* images */
+
+.img--container{
+    border: 1px solid var(--GRAY-80);
+    padding: 1rem;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
```
Comment (theVivek-kumar) [inline image/image.css:9]: class name is good

File: image/image.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    <li class="aside--item">
+                        <a href="#" class="aside--link">Slider</a>
+                    </li>
+                    <li class="aside--item">
+                        <a href="#" class="aside--link">Drawer</a>
+                    </li>
+                </ul>
+            </div>
+
+        </aside>
```
Comment (theVivek-kumar) [inline image/image.html:126]: this looks good to me

**Review by theVivek-kumar — APPROVED** [review]
looks good to me

**Pull request merged**